src/uct/ib/base/ib_iface.c

```diff
@@ -650,6 +651,11 @@ uct_ib_iface_roce_is_reachable(const uct_ib_device_gid_info_t *local_gid_info,
         ucs_debug("different addr_family detected. local %s remote %s",
                   ucs_sockaddr_address_family_str(local_ib_addr_af),
                   ucs_sockaddr_address_family_str(remote_ib_addr_af));
+        uct_iface_populate_info_str_buf(
```

Comment:
can we move the ucs_debug message inside uct_iface_populate_info_str_buf function? since essentially they are printing the same thing